client/components/command-palette/use-command-palette.tsx

```diff
@@ -95,28 +114,32 @@ const useSiteToAction = () => {
 					subLabel: `${ site.URL }`,
 					searchLabel: `${ site.ID } ${ siteName } ${ site.URL }`,
 					callback: ( { close } ) => {
-						dispatch(
-							recordTracksEvent( 'calypso_hosting_command_palette_site_select', {
-								command: selectedCommand.name,
-								list_count: filteredSitesLength,
-								list_visible_count: listVisibleCount,
-								current_route: currentRoute,
-								search_text: search,
-								command_site_id: site.ID,
-								command_site_has_custom_domain: isCustomDomain( site.slug ),
-								command_site_plan_id: site.plan?.product_id,
-							} )
-						);
+						recordTracksEvent( 'calypso_hosting_command_palette_site_select', {
+							command: selectedCommand.name,
+							list_count: filteredSitesLength,
+							list_visible_count: listVisibleCount,
+							current_route: currentRoute,
+							search_text: search,
+							command_site_id: site.ID,
+							command_site_has_custom_domain: isCustomDomain( site.slug ),
+							command_site_plan_id: site.plan?.product_id,
+						} );
 						onClickSite( { site, close, command: selectedCommand } );
 					},
 					image: (
 						<FillDefaultIconWhite>
-							<SiteIcon site={ site } size={ 32 } />
+							{ site.icon?.img ? (
```

Comment:
Can we have sites without the icon property? The icon comes from a value in an array of strings that doesn't enforce existence.

Reply:
Yes, there are sites without icon.